app/src/main/res/layout/activity_main.xml

```diff
@@ -85,11 +133,46 @@
             android:layout_height="30dp"
             android:layout_margin="10dp"
             android:background="@android:color/transparent"
-            mapbox:srcCompat="@drawable/speaker" />
+            mapbox:srcCompat="@drawable/speaker"
+            tools:ignore="SpeakableTextPresentCheck,TouchTargetSizeCheck"
+            android:importantForAccessibility="no" />
     </LinearLayout>
 
     <include
         layout="@layout/incident_form_fragment"
         mapbox:layout_anchor="@+id/mapView"
         mapbox:layout_anchorGravity="center" />
+
+    <LinearLayout
+        android:id="@+id/lateralMenu"
+        android:layout_width="300dp"
+        android:layout_height="match_parent"
+        android:background="@color/blueOxford_app_theme"
+        android:elevation="10dp"
+        android:gravity="bottom"
+        android:orientation="vertical"
+        android:visibility="gone"
+        mapbox:layout_constraintBottom_toBottomOf="parent"
+        mapbox:layout_constraintStart_toStartOf="parent"
+        mapbox:layout_constraintTop_toTopOf="parent">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="60dp"
+            android:gravity="center"
+            android:onClick="showAccount"
+            android:text="Account"
```

Comment:
This text should be in a string within a resource xml file.

Reply:
Done